blocks/procedures.js

```diff
@@ -737,14 +738,15 @@ const procedureDefMutator = {
     }
     this.updateParams_();
     Procedures.mutateCallers(this);
-    for (let i = this.model.getParameters().length; i >= 0; i--) {
-      this.model.deleteParameter(i);
+
```

Comment:
Optional: maybe store the procedure model in a var here vs calling the getter a bunch?